ruby-challenges.rb

```diff
@@ -33,4 +62,29 @@
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
 
-# Pseudo code:
\ No newline at end of file
+
+# Pseudo code
+# I made a function pal with a str parameter
+def pal str
+    
+#Made a conditional statement that used .reverse to reverse my string and would return whatever is a palindrone and retunr whatever was not a palindrome
+        if str.reverse == str
+            "#{str} is a palindrome"
+        else
+            "#{str} is not a palindrome"  
+    
+        end
+end 
+#Print the test variables and try to get expected outcome
+p pal palindrome_test_case1 
+p pal palindrome_test_case2 
+p pal palindrome_test_case3 
+#outcome I kept getting:
+# "Racecar is not a palindrome"
+# "LEARN is not a palindrome"
+# "Rotator is not a palindrome"
+
+#I worked on my code for like an hour and couldnt understand why I wasnt getting the expected output I wanted. I got frustrated and went and found the solution on Chat. 
+
+
+# str = str.downcase <-- found the fix on Chat
```

Comment:
Yep!  Same for the one above with the album names.  You need to account for both cases.  Good work logically!